Library/Homebrew/os/mac.rb

```diff
@@ -37,6 +37,15 @@ def outdated_release?
       version < "10.9"
     end
 
+    def os_name(version = @version)
+      # TODO: replace os_name by constant string "macOS" when macOS 10.14 is released
+      if Version.create(version) >= Version.create("10.12")
```

Comment:
Check the methods above; if you create a version the same as them (probably with `.new`) then you should be able to compare the same as them.